src/components/Task.vue

```diff
@@ -0,0 +1,82 @@
+<template>
+  <li :class="{done: task.taskState}">
+    <div>
+      <input
+        type="checkbox"
+        v-model="task.taskState"
+        @change="clickOnCheckbox"
+      >
+      <span class="title">
+        {{task.taskText}}
+      </span>
+    </div>
+    <button
+      class="item-btn"
+      @click="onClickDeltTaskBtn"
+    >Delete</button>
+  </li>
+</template>
+
+<script>
+export default {
+  name: 'Task',
+  props: ['task', 'listId'],
+  methods: {
+    onClickDeltTaskBtn() {
+      this.$emit('onClickDeltTaskBtn1', {
+        taskId: this.task.taskId,
+        listId: this.listId,
+      });
+    },
+    clickOnCheckbox() {
+      this.btnDisable = !this.btnDisable;
```

Comment:
I don't see definition or using **btnDisable** property, looks like it remains from the previous version of the code